R/example.R

```diff
@@ -484,33 +500,38 @@ example_jira_two_issues <- function(folder_path="/tmp",folder_name) {
 #' @return the JSON path of the newly created issue issue tracker
 #' @export
 #' @keywords internal
-example_jira_issue_comments <- function(folder_path="/tmp",folder_name) {
+example_jira_issue_comments <- function(folder_path, folder_name) {
 
   # Create folder & repo
-  folder_path <- io_make_folder(folder_path=folder_path, folder_name = folder_name)
-
-  issue1 <- make_jira_issue(jira_domain_url = "https://project.org/jira",
-                            issue_key = "GERONIMO-2",
-                            issue_type = "Bug",
-                            status = "In Progress",
-                            resolution = "Incomplete",
-                            title = "Bug fixes need implementation.",
-                            description = "The new features have not been implemented.",
-                            components = "x-core",
-                            creator_name = "Moe",
-                            reporter_name = "Larry",
-                            assignee_name = "Curly",
-                            comments = c(
-                              "This is the first body comment.",
-                              "This is the second body comment."
-                            )
+  folder_path <- io_make_folder(folder_path = folder_path, folder_name = folder_name)
+
+  issue1 <- make_jira_issue(
+    jira_domain_url = "https://project.org/jira",
+    issue_key = "GERONIMO-123",
+    project_key = "GERONIMO",
+    summary = "Summary of new feature",
+    description = "The new features have been implemented",
+    issue_type = "New Feature",
+    resolution = "Finished",
+    priority = "Minor",
+    status = "Open",
+    labels = c("pull-request-available"),
+    components = c("jira", "mail"),
+    affects_versions = c("3.4.3"),
+    fix_versions = c("3.4.2"),
+    assignee_name = "Moe",
+    creator_name = "Bob",
+    reporter_name = "Joe",
+    comments = c(
+      "This is the first body comment.",
+      "This is the second body comment."
+    )
   )
 
-  # issues <- c(list(issue1))
   issues <- list(issue1)
 
-  jira_json_path <- make_jira_issue_tracker(issues,
-                                            save_filepath=file.path(folder_path,"one_issue_two_comments.json"))
+  jira_json_path <- make_jira_issue_tracker(
+    issues, save_filepath=file.path(folder_path,"SPARK_issues_1121646814_1121719175.json"))
```

Comment:
Same issue with identifying actual projects. Make sure no other identifying information is being versioned into the code.